modules/extension/wms/src/main/java/org/geotools/data/wms/request/AbstractGetStylesRequest.java

```diff
@@ -34,11 +35,10 @@
 
     /**
      * @param onlineResource
-     * @param layers
+     * @param properties
      */
-    public AbstractGetStylesRequest( URL onlineResource, Layer[] layers) {
-        super(onlineResource, null);
-        this.layers = layers;
+    public AbstractGetStylesRequest( URL onlineResource, Properties properties ) {
+        super(onlineResource, properties);
     }
```

Comment:
I am having some problems with this public API change, you have orphaned the private Layer[] layers field (and the getLayers() method that refers to it).

This is making my review go slower :(